releasing/images/general_context.svg

```diff
@@ -0,0 +1,4985 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do we need both the png and the svg (i.e. can't we use the svg directly on the page?)

Reply:
No idea if they are going to be rendering nicely in the ignition site. Using directly the SVG would be great, we can try. There is some examples of what is working using github for SVG here https://github.com/potherca-blog/StackOverflow/tree/master/question.13808020.include-an-svg-hosted-on-github-in-markdown . Do you have any suggestion about which one to use?

Reply:
use SVG directly and remove png af80e93